runners/apex/src/main/java/org/apache/beam/runners/apex/ApexRunner.java

```diff
@@ -72,6 +78,12 @@
    */
   public static final AtomicReference<AssertionError> ASSERTION_ERROR = new AtomicReference<>();
 
+  /**
+   * The runner will look for this file in the classpath for Apex configuration properties
+   * unless an alternative location was specified in pipeline options.
+   */
+  public static final String RUNNER_PROPERTIES_RESOURCE = "/beam-runners-apex.properties";
```

Comment:
Updated with explicit URL scheme for classpath and made the default visible through the options.